In [1]:
import numpy as np

In [2]:
discounts = {}
discounts['January'] = 00.5
discounts['February'] = 1.3
discounts['March'] = 3.5
discounts['April'] = 10
discounts['May'] = 20
discounts['June'] = 5
discounts['July'] = 9
discounts['August'] = 1
discounts['September'] = 1
discounts['October']=1
discounts['November'] =0
discounts['December']=0
discounts

{'January': 0.5,
 'February': 1.3,
 'March': 3.5,
 'April': 10,
 'May': 20,
 'June': 5,
 'July': 9,
 'August': 1,
 'September': 1,
 'October': 1,
 'November': 0,
 'December': 0}

In [20]:
months = discounts.keys()
prices = {}

In [22]:
for m in months:
    prices[m] = float(input())
prices

100
110
120
150
100
120
130
90
130
170
200
235


{'January': 100.0,
 'February': 110.0,
 'March': 120.0,
 'April': 150.0,
 'May': 100.0,
 'June': 120.0,
 'July': 130.0,
 'August': 90.0,
 'September': 130.0,
 'October': 170.0,
 'November': 200.0,
 'December': 235.0}

##### January	100
February	110
March	120
April	150
May	100
June	120
July	130
August	90
September	130
October	170
November	200
December	235
